### PR TITLE
Updated to 1.609.1.1 plugin versions

### PR DESCRIPTION
**Pull request by jglick**
Testing:
- [X] Installing in “license only” mode and then getting an evaluation license
- [x] Installing in “full” mode and then getting a Team Edition license
- [x] Installing in “full” mode and then getting an Enterprise Edition license
- [x] Installing in “OC-client only” mode and then connecting to a CJOC server with a Team Edition license

@reviewbybees (ref. CJE-2213) esp. @stephenc

### CODE REVIEW

File: src/main/java/com/cloudbees/jenkins/plugins/enterpriseplugins/PluginImpl.java [hunk 1]
```diff
@@ -93,105 +93,117 @@
...
-            require("cloudbees-groovy-view","1.4"),
-            OC_AGENT, OC_CONTEXT, OC_CLIENT, OC_CLOUD, OC_OPENID_CSE, OC_ANALYTICS,
-            require("cloudbees-workflow-aggregator", CLOUDBEES_WORKFLOW_VERSION),
-            require("workflow-cps-checkpoint", CLOUDBEES_WORKFLOW_VERSION),
-            require("cloudbees-workflow-rest-api", CLOUDBEES_WORKFLOW_VERSION),
-            require("cloudbees-workflow-ui", CLOUDBEES_WORKFLOW_VERSION)
+            require("ssh-agent","1.7"),
+            require("ssh-credentials","1.11"),
+            require("ssh-slaves","1.9"),
+            require("support-core","2.25"),
```
Comment (stephenc) [inline src/main/java/com/cloudbees/jenkins/plugins/enterpriseplugins/PluginImpl.java]: duplicate

File: src/main/java/com/cloudbees/jenkins/plugins/enterpriseplugins/PluginImpl.java [hunk 2]
```diff
@@ -93,105 +93,117 @@
...
+            require("cloudbees-jsync-archiver","5.4"),
+            require("cloudbees-label-throttling-plugin","3.4"),
+            require("cloudbees-long-running-build","1.4"),
+            require("cloudbees-monitoring","1.7"),
+            require("cloudbees-nodes-plus","1.11"),
+            require("cloudbees-plugin-usage","1.5"),
+            require("cloudbees-quiet-start","1.1"),
+            require("cloudbees-secure-copy","3.7"),
+            require("cloudbees-ssh-slaves","1.2"),
+            require("cloudbees-support","3.3"),
```
Comment (stephenc) [inline src/main/java/com/cloudbees/jenkins/plugins/enterpriseplugins/PluginImpl.java]: duplicate

**Comment by jglick** [comment]
OK, CJOC 1.7.0 is on the update center now, so will recheck that.

**Comment by oleg-nenashev** [comment]
:+1:

**Comment by jtnord** [comment]
:+1:

**Pull request merged**